### ISSUE TITLE
Searchbox accessibility error

### ISSUE THREAD

**Issue by jfrank-nih** [NCIOCPL/gatsby-pcp#128]
This searchinput element does not have a name available to an accessibility
   API. Valid names are: label element, title undefined, aria-label undefined,
   aria-labelledby undefined.

   (#searchbox-header-input)

   <input class="usa-input" id="searchbox-header-input" type="search"
   name="swKeyword" data-autosuggest-collection="cgov">
